src/Components/Settings/AdminComponent.jsx

```diff
@@ -140,6 +155,16 @@ const AdminComponent = () => {
                     Toast.success({
                         text: "Role Changed Successfully",
                     });
+
+                    let changedUser = adminData?.filter((user) => user.userId == userId);
+                    console.log(changedUser)
+                    //////////////////////////
```

Comment:
Role changed

Reply:
Added a function `updateUserPermission: async (UserId, UserRole) => {` to generalFunction